frigg_worker/jobs.py

```diff
@@ -84,7 +84,7 @@ def succeeded(self):
     def settings(self):
         return build_settings(self.working_directory, self.docker)
 
-    def clone_repo(self, depth=1):
+    def clone_repo(self, depth=20):
```

Comment:
Right now it would fail, but we might clone without depth if it is not high enough? It shouldn't happen to often if we adjust the depth as we se how often it happens.